console_conf/controllers/identity.py

```diff
@@ -194,9 +194,9 @@ def write_login_details_standalone():
 
 class IdentityController(BaseController):
 
-    def __init__(self, common):
-        super().__init__(common)
-        self.model = self.base_model.identity
+    def __init__(self, app):
+        super().__init__(app)
+        self.model = app.base_model.identity
```

Comment:
This is change of behaviour =) self -> common (now app)

Reply:
Ehhhh

Reply:
that was a joke =)